ui/admin/app/components/form/host-catalog/gcp/index.hbs

```diff
@@ -240,10 +240,36 @@
     {{/if}}
   </Hds::Form::Textarea::Field>
 
+  <Hds::Form::Textarea::Field
+    name='worker_filter'
+    @value={{@model.worker_filter}}
+    @isOptional={{true}}
+    @isInvalid={{@model.errors.worker_filter}}
+    disabled={{form.disabled}}
+    as |F|
+  >
+    <F.Label>{{t 'form.worker_filter.label'}}</F.Label>
+    <F.HelperText>
+      {{t 'resources.host-catalog.form.worker_filter.help'}}
+      <Hds::Link::Inline @href={{doc-url 'worker.create-tags'}}>
+        {{t 'actions.learn-more'}}
+      </Hds::Link::Inline>
+    </F.HelperText>
+    {{#if @model.errors.worker_filter}}
+      <F.Error as |E|>
+        {{#each @model.errors.worker_filter as |error|}}
+          <E.Message>
+            {{error.message}}
+          </E.Message>
+        {{/each}}
+      </F.Error>
+    {{/if}}
+  </Hds::Form::Textarea::Field>
+
   <Hds::Form::Checkbox::Field
     name='disable_credential_rotation'
     checked={{@model.disable_credential_rotation}}
-    readonly={{form.disabled}}
+    disabled={{form.disabled}}
```

Comment:
I changed this to disabled because I was able to interact with the field when the form was in a non-edit state

Reply:
Good call. I think we will have to keep doing that until we find a both forward for checkboxes. I reached out to the design system team to figure out how we should handle these.

Reply:
awesome, thank you!

Reply:
Good call!